utest/api/test_logging_api.py

```diff
@@ -79,6 +79,13 @@ def test_logged_to_python(self):
         logger.write("Doo", 'INFO')
         assert_equal(self.handler.messages, ['Foo', 'Boo', 'Goo', 'Doo'])
 
+    def test_empty_string_not_looged_to_python(self):
```

Comment:
Could you fix the typo? (It says looged instead of logged)

Reply:
Done!